examples/install/online-boutique/gloo-mesh-config/workspaces.yaml

```diff
@@ -70,10 +71,6 @@ spec:
     - selector:
```

Comment:
Can we get rid of this and use the default? https://docs.solo.io/gloo-mesh-enterprise/latest/reference/api/github.com.solo-io.gloo-mesh-enterprise.api.gloo.solo.io.admin.v2.workspace_settings/#workspacesettingsspecoptions
`Selects the ingress gateways in a workspace. If no gateways are selected, defaults to attempting to use istio-eastwestgateway.istio-system as the Istio ingress gateway for “east-west” traffic`

Reply:
Nevermind, default only looks in istio-system